apps/cmp.c

```diff
@@ -643,10 +643,6 @@ static X509 *load_cert_pwd(const char *uri, const char *pass, const char *desc)
     return cert;
 }
 
-/*
- * TODO potentially move this and related functions to apps/lib/
- * or even better extend OSSL_STORE with type OSSL_STORE_INFO_CRL
```

Comment:
Interesting to see that idea over a function that's *not* about CRLs...

Reply:
When writing this commend about `load_csr_autofmt()` I confused CSR and CRL; I intended to write:
```
TODO potentially move this and related functions to apps/lib/
or even better extend OSSL_STORE with type OSSL_STORE_INFO_CSR
```

Reply:
Would it make sense to extend OSSL_STORE this way?

Reply:
Perhaps it would however not at this time.

Reply:
The FR for this is in  #15725.